### ISSUE TITLE
Is `ember-runtime-enumerable-includes-polyfill` dependency still needed?

### ISSUE THREAD

**Issue by psteininger** [kaliber5/ember-bootstrap#579]
This dependency causes compile-time deprecation warning on ember-cli 2.18.x. It seems like the dependency is unmaintained and possibly not needed any more

DEPRECATION: An addon is trying to access project.nodeModulesPath. This is not a reliable way to discover npm modules. Instead, consider doing: require("resolve").sync(something, { basedir: project.root }). Accessed from:   new NPMDependencyVersionChecker (/<project_root>/node_modules/ember-runtime-enumerable-includes-polyfill/node_modules/ember-cli-version-checker/src/npm-dependency-version-checker.js:11:33)

**Comment by simonihmig** [comment]
Yes, I think it is still needed for Ember versions that don't have `Ember.Array#includes` yet. 
The deprecation is actually thrown in `ember-cli-version-checker`, but this addon has been fixed already. Maybe you have older versions in your `yarn.lock` and just need to `yarn upgrade`? (or the npm equivalent)

**Comment by psteininger** [comment]
@simonihmig I have been hunting down that deprecation, being concerned that if I go for Ember 3.x things will break all over the place. sadly I only get one source of the issue at a time. It is a problem with the older version of `ember-cli-version-checker` indeed. I just thought that if `ember-runtime-enumerable-includes-polyfill`  hasn't been touched in over a year, it may no longer be needed. Perhaps a dependency update there could be all that's needed.

**Comment by simonihmig** [comment]
But the latest release already references the latest version of `ember-cli-version-checker`: https://github.com/rwjblue/ember-runtime-enumerable-includes-polyfill/blob/master/package.json#L50. That's why I guessed that you might have stale dependencies, and just need to update them.

And for this simple polyfill, there is probably no need for any other updates. But as I said it is needed for older Ember versions.

**Comment by psteininger** [comment]
@simonihmig I think the problem lies on this line: https://github.com/rwjblue/ember-runtime-enumerable-includes-polyfill/blob/master/package.json#L27. I submitted a PR for it.
